utils/schema.js

```diff
@@ -0,0 +1,81 @@
+import { ApolloServer, gql } from 'apollo-server-micro';
```

Comment:
You should have a separate file for typeDefs and another one for resolvers. Probably put them in utils/graphql instead of just utils like I requested to keep things clean.